lib/stream-chat/client.rb

```diff
@@ -303,6 +303,10 @@ def send_file(relative_url, file_url, user, content_type = 'application/octet-st
       parse_response(response)
     end
 
+    def check_sqs(sqs_key = nil, sqs_secret = nil, sqs_url = nil)
```

Comment:
`sqs_` prefix seems redundant; key, secret and url would be sufficient. wdyt ?

Reply:
I tried to match them to the params of the request, but I agree. I can change it, will have to do it in the other sdk's as well.

Reply:
Yup, I understand but I am thinking from the client view. I am fine with both actually.

Reply:
I don't feel strongly about it, but at least this way they can't be mixed up with other credentials. I'll leave it like this then.

Reply:
Sounds good.